src/pages/index.tsx

```diff
@@ -1,72 +1,79 @@
-import type { NextPage } from 'next'
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import type { NextPage } from "next";
+import Head from "next/head";
+import Image from "next/image";
+import styles from "../styles/Home.module.scss";
+import TestComponent from "../components/TestComponent";
+import Calendar from "react-calendar";
+import "react-calendar/dist/Calendar.css";
+import React, { useEffect } from "react";
+import { useState } from "react";
+import { languages } from "../../data/expConst";
+import Select from "react-select";
 
 const Home: NextPage = () => {
+  //define our states so we can access the data the user types
+  const [value, onChange] = useState(new Date());
+  const [language, onChangeLang] = useState();
+  var [firstnameINPUT, onFirstnameCHANGE] = useState("");
```

Comment:
And unintuitively, these states should use const.   Since react controls these variables, we will never change them manually, thus const in extra protection against us modifying the values outside of the setter callback.